.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -53,6 +54,49 @@ Bugs & improvements:
 
 - [ ] (1) ...
 
+_____
+
+## Patch Release
+
+### TASKS:
+
+ - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from `latest`
+ - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
+ - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
+ - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder with the proper fixes
+ - [ ] [DIS] Move the `unreleased` folder from `master` to `changelog` folder, to avoid Calens' conflicts problems
+ - [ ] [DEV] Add release notes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources, in `ReleaseNotesViewModel.kt` (in case patch release before OEM)
+ - [ ] [DIS] Check for new screenshots in Play Store / GitHub repo and generate them
+ - [ ] [DIS] Generate final bundle from last commit in owncloud/android
+ - [ ] [DEV] Code Review
+ - [ ] [QA] Test execution
+ - [ ] [QA] Trigger BitRise builds for unit tests and UI tests, in case CHANGELOG conflicts avoid them in GitHub
+ - [ ] [QA] QA Approval
+ - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
+ - [ ] [DIS] Upload & publish release bundle and changelog in Play Store
+ - [ ] [DIS] Update screenshots and store listing, if needed, in Play Store
+ - [ ] [GIT] Create and sign tag `vM.m.p` in HEAD commit of release branch, in owncloud/android
+ - [ ] [GIT] Move tag `latest` pointing the same commit as the release commit
+ - [ ] [GIT] Publish a new [release](https://github.com/owncloud/android/releases) in owncloud/android
+ - [ ] [DIS] Release published in Play Store
+ - [ ] [COM] Inform `#updates` in internal chat that release is out
+ - [ ] [GIT] Fix conflicts if they happen. GitHub option will help and will merge `master` into release branch, getting it ready to fast forward
```

Comment:
those two things are different. After signing the tag and so on, the final target is merging the release branch into `master`. We will be allowed to do this if release branch is updated with `master`. But, as we agreed, we will not rebase release branches anymore to avoid differences in the history, so , the only way to update the release branch with `master` is merging `master` into release branch fixing conflicts if there are. Github provides options to do this.